src/app/pages/jails/jail-add/jail-add.component.ts

```diff
@@ -118,538 +127,644 @@ export class JailAddComponent implements OnInit {
+                 <i>2</i>.'),
+    },
+    {
       type: 'checkbox',
       name: 'allow_mount_tmpfs',
-      placeholder: T('allow_mount_tmpfs'),
-      tooltip: T('Check to allow privileged users inside the jail to mount\
- and unmount the tmpfs file system. This permission is effective only\
- together with <b>allow_mount</b> and if <b>enforce_statfs</b> is set to\
- a value lower than <i>2</i>.'),
-    }, {
+      placeholder: T('allow.mount.tmpfs'),
+      tooltip: T('Set to allow privileged users inside the jail to mount \
+                 and unmount the tmpfs file system. This permission is \
+                 only effective when <b>allow_mount</b> and \
+                 <b>enforce_statfs</b> are set to a value lower than \
+                 <i>2</i>.'),
+    },
+    {
       type: 'checkbox',
       name: 'allow_mount_zfs',
-      placeholder: T('allow_mount_zfs'),
-      tooltip: T('Check to allow privileged users inside the jail to mount\
- and unmount the ZFS file system. This permission is effective only\
- together with <b>allow_mount</b> and if <b>enforce_statfs</b> is set to\
- a value lower than <i>2</i>.'),
-    }, {
+      placeholder: T('allow.mount.zfs'),
+      tooltip: T('Set to allow privileged users inside the jail to mount \
+                 and unmount the ZFS file system. This permission is \
+                 only effective when <b>allow_mount</b> and \
+                 <b>enforce_statfs</b> are set to a value lower than \
+                 <i>2</i>. The <a \
+href="https://www.freebsd.org/cgi/man.cgi?query=zfs&sektion=8&apropos=0&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
+target="_blank">ZFS(8)</a> man page has information on how to configure \
+                 the ZFS filesystem to operate from within a jail.'),
+    },
+    {
       type: 'checkbox',
       name: 'allow_quotas',
-      placeholder: T('allow_quotas'),
-      tooltip: T('Check to allow the jail root to administer quotas on the\
- jail filesystems. This includes filesystems the jail may share with\
- other jails or with non-jailed parts of the system.'),
-    }, {
+      placeholder: T('allow.quotas'),
+      tooltip: T('Set to allow the jail root to administer quotas on \
+                 jail filesystems. This includes filesystems the jail \
+                 may share with other jails or with non-jailed parts \
+                 of the system.'),
+    },
+    {
       type: 'checkbox',
       name: 'allow_socket_af',
-      placeholder: T('allow_socket_af'),
-      tooltip: T('Check to allow access to other protocol stacks beyond\
- IPv4, IPv6, local (UNIX), and route. <b>Warning:</b> jail functionality\
- may not exist for other protocal stacks.'),
+      placeholder: T('allow.socket_af'),
+      tooltip: T('Set to allow access to other protocol stacks beyond \
+                  IPv4, IPv6, local (UNIX), and route. <br>\
+                  <b>Warning:</b> jail functionality may not exist for \
```

Comment:
See above.

Reply:
fixed
jail functionality may not exist for all protocol stacks
jail functionality does not exist for all protocol stacks